core/includes/common.inc

```diff
@@ -3451,8 +3451,8 @@ function backdrop_css_defaults($data = NULL) {
  *       which backdrop_add_css() happened earlier in the page request.
  *   - 'media': The media type for the stylesheet, e.g., all, print, screen.
  *     Defaults to 'all'.
- *   - attributes: An associative array of attributes for the <link> tag.
- *     This may be used to add 'integrity' or custom attributes. Note that
+ *   - attributes: (optional) An associative array of attributes for the <link>
```

Comment:
I don't think this edit is needed. It already says that the $options array is optional which includes this.

Reply:
I'll defer to the prevailing standards, of course, but I see "(optional)" used in this way in lots of places in common.inc for optional arguments and parameters.

Reply:
I agree with @herbdool here. Otherwise we'd need to add `(optional)` to the rest of the `$options`.

Reply:
OK, will update.